docker/web/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Pedro Marques <roque@juniper.net>
+RUN apt-get install -y software-properties-common
+RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
+RUN apt-get update
+RUN apt-get -y install gdebi wget
+
+RUN wget -qO /root/nodejs_0.8.15-1contrail1_amd64.deb https://launchpad.net/~opencontrail/+archive/ubuntu/ppa/+files/nodejs_0.8.15-1contrail1_amd64.deb
+RUN gdebi -n /root/nodejs_0.8.15-1contrail1_amd64.deb
```

Comment:
Got it. I was not able to get the right version to search
(nodejs=0.8.15-1contrail1). I was using just nodejs=0.8.15. I have updated
the pull request.

Thanks

On Thu, Oct 29, 2015 at 5:16 AM, Pedro Marques <notifications@github.com>
wrote:

> In docker/web/Dockerfile
> <https://github.com/pedro-r-marques/examples/pull/1#discussion_r43333875>:
>
> > @@ -0,0 +1,29 @@
> > +FROM ubuntu:14.04
> > +MAINTAINER Pedro Marques <roque@juniper.net>
> > +RUN apt-get install -y software-properties-common
> > +RUN apt-add-repository -y ppa:anantha-l/opencontrail-2.20
> > +RUN apt-get update
> > +RUN apt-get -y install gdebi wget
> > +
> > +RUN wget -qO /root/nodejs_0.8.15-1contrail1_amd64.deb https://launchpad.net/~opencontrail/+archive/ubuntu/ppa/+files/nodejs_0.8.15-1contrail1_amd64.deb
> > +RUN gdebi -n /root/nodejs_0.8.15-1contrail1_amd64.deb
>
> you are downloading the package manually vs using apt-get download; that
> doesn't seem required.
> And then using an extra tool for the install of the dependent package...
> what problem does it solve ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pedro-r-marques/examples/pull/1/files#r43333875>.
>